src/Akeneo/Pim/Automation/DataQualityInsights/front/src/application/component/ProductEditForm/TabContent/DataQualityInsights/AttributesGroupsHelper.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styled from 'styled-components';
+import {useCatalogContext, useProductEvaluatedAttributeGroups} from "../../../../../infrastructure/hooks";
+import {AttributeGroup} from "@akeneo-pim-community/settings-ui/src/models";
+
+const translate = require('oro/translator');
+
+const Helper = styled.div`
+    margin: 5px 0 10px 0;
+    background-color: #f5f9fc;
+    padding: 13px 13px 13px 0;
+    display: flex;
+`;
+
+const HelperIcon = styled.div`
+    padding: 0 22px 0 22px;
+    margin-right: 22px;
+    border-right: 1px #c7cbd4 solid;
+    background-image: url(/bundles/pimui/images/icon-info.svg);
+    background-repeat: no-repeat;
+    background-size: 20px 20px;
+    background-position: center;
+    width: 64px;
+`;
+
+const AttributeGroupsHelper = () => {
+  const evaluatedAttributeGroups = useProductEvaluatedAttributeGroups();
+  const {locale} = useCatalogContext();
+
+  if (!evaluatedAttributeGroups || Object.keys(evaluatedAttributeGroups).length === 0 || locale === undefined) {
+    return (<></>);
+  }
+
+  return (
+    <Helper>
+      <HelperIcon/>
+      <div>
+        {translate('akeneo_data_quality_insights.attribute_group.used_groups_helper')}&nbsp;
+        <span data-testid='dqi-evaluated-attribute-groups'>
+          {Object.entries(evaluatedAttributeGroups)
+            .map(([_, group]:[string, AttributeGroup]) => group.labels[locale])
+            .join(', ')
+          }.
+        </span>
+      </div>
+    </Helper>
```

Comment:
I think you can use the HelperRibbon component: https://github.com/akeneo/pim-community-dev/blob/master/src/Akeneo/Platform/Bundle/UIBundle/Resources/workspaces/shared/src/components/HelperRibbon.tsx

```suggestion
    <HelperRibbon level={HelperLevel.HELPER_LEVEL_INFO}>
      {translate('akeneo_data_quality_insights.attribute_group.used_groups_helper')}&nbsp;
      <span data-testid='dqi-evaluated-attribute-groups'>
        {Object.entries(evaluatedAttributeGroups)
          .map(([_, group]:[string, AttributeGroup]) => group.labels[locale])
          .join(', ')
        }.
      </span>
    </HelperRibbon>
```

Reply:
I checked and I can't : the render of this component is not the one Lucas wants (I checked with him). But he'll ask the team that created and use it to update it. In the meantime, I'll keep my component.